src/Stateless/Stateless.csproj

```diff
@@ -1,29 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Create state machines and lightweight state machine-based workflows directly in .NET code</Description>
     <AssemblyTitle>Stateless</AssemblyTitle>
+    <AssemblyName>Stateless</AssemblyName>
+    <Product>Stateless</Product>
+    <TargetFrameworks>netstandard2.0;netstandard1.0;net45;net40</TargetFrameworks>
```

Comment:
I'd strongly suggest adding `net461` and `net472` targets in addition, to these to avoid issues for people targettng those frameworks [as described in the docs](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting). 

The tl;dr; is that MS screwed up back-porting netstandard2.0 support to net461, and adding those extra targets avoids the issues

Reply:
Thanks 👍

Reply:
please make an additional PR for this.
as i already removed my fork+branch, not sure how to make new commits to this PR now...